mudirac/mudirac.xml

```diff
@@ -185,9 +205,24 @@ spec_expdec: $spec_expdec
             <option value="SPHERE"> finite size, uniformly charged spherical nucleus </option>
             <option value="FERMI2"> Fermi 2-term charge distribution </option>
         </param>
-        <param name="uehling_correction" type="boolean" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not" optional="true"/>
+        <conditional name="uehling_correction">
+            <param name="uehling_correction" type="select" display="radio" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not">
+                <option value="false" selected="true">False</option>
+                <option value="true">True</option>
+            </param>
+            <when value="false"/>
+            <when value="true">
+                <param name="uehling_steps" type="integer" value="" label="integration steps for the Uehling potential." help=" Higher numbers will make the Uehling energy more precise but increase computation times. Default is 100." optional="true"/>
+                <param name="uehling_lowcut" type="float" value="" label="low cutoff for Uehling potential" help="under low cutoff, the radius will be considered 0. Default is 0." optional="true"/>
```

Comment:
```suggestion
                <param name="uehling_lowcut" type="float" value="0" min="0" label="low cutoff for Uehling potential" help="under low cutoff, the radius will be considered 0. Default is 0."/>
```
A radius should not be negative.

May be associated with the following in the error log (I changed a lot of things to be negative at once to try and break the tool):
```
2024-07-01 14-18-07.979 [Err] (main) Unknown error: State is not converged
```